content/concepts/dolthub/api.md

```diff
@@ -313,3 +313,166 @@ res = requests.get(
 with open(local_file, 'wb') as file:
     file.write(res.content)
 ```
+
+## Database API
+
+### Create a database
+
+Creates a new database. Python sample code:
+
+```python
+import requests
+data = {"description": "database description", "ownerName": "dolthub",
+        "repoName": "new-database-name", "visibility": "public"}
```

Comment:
These examples would be better if you used an example of an actual database you'd create so you can tell the story of someone who would use all these api endpoints (as opposed to just placeholders). For example, can use the `museum-collections` database and tell the story of "I'm creating this database `museum-collections` where I'm going to aggregate all records from museums around the world, use the create database endpoint to create it, the SQL API to add some data on a new branch,  create a pull request with data from x museum so that someone can review my changes on dolthub, can respond to comments, and then I can merge"